news/migrations/0017_use_datetime_instead_of_date_and_time.py

```diff
@@ -0,0 +1,124 @@
+# Generated by Django 2.2.8 on 2020-02-02 11:53
+from datetime import datetime
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+def merge_article_date_and_time(apps, schema_editor):
+    Article = apps.get_model("news", "Article")
+    for article in Article.objects.all():
+        article.publication_time = datetime.combine(article.pub_date, article.pub_time)
+        article.save()
+
+
+def reverse_merge_article_date_and_time(apps, schema_editor):
+    Article = apps.get_model("news", "Article")
+    for article in Article.objects.all():
+        article.pub_date = article.publication_time.date()
+        article.pub_time = article.publication_time.time()
+        article.save()
+
+
+def merge_timeplace_date_and_time(apps, schema_editor):
+    Timeplace = apps.get_model("news", "Timeplace")
+    for timeplace in Timeplace.objects.all():
+        timeplace.publication_time = datetime.combine(timeplace.pub_date, timeplace.pub_time)
+        timeplace.start_time = datetime.combine(timeplace.start_date, timeplace.temp_start_time)
+        timeplace.end_time = datetime.combine(
+            timeplace.end_date if timeplace.end_date is not None else timeplace.start_date, timeplace.temp_end_time)
```

Comment:
Curious why `start_date` and `temp_end_time` are combined when there is no `end_date`.. Because it's the closest we can get to a datetime that seems about right..? 🤔

Reply:
From what I remember when Amund implemented this 3 years ago was that `end_date` would be `None` only when the start and end date was the same. Thus I have chosen to combine `start_date` with `temp_end_time` in those cases.